curvefs/src/metaserver/heartbeat.cpp

```diff
@@ -254,15 +254,16 @@ int Heartbeat::BuildRequest(HeartbeatRequest* req) {
         return -1;
     }
 
-    req->set_metadataspaceused(capacityKB - availKB);
-    req->set_metadataspacetotal(capacityKB);
+    // TODO(cw123) : this should replace
```

Comment:
The MetaserverSpaceState info set here later by Jingli Chen?

Reply:
yes